Initial change for changeLog and readme (#5658)
Azure/azure-sdk-for-java: sdk/storage/azure-storage-queue/CHANGELOG.md

# Release History

> ## Version 12.0.0-preview.4 (2019-10-8)

Reviewer: Same feedback applies to this file
1. update Java Beans and Module support text to match the other 2 changelogs
2. quote `code` in this file to be consistent with the blobs changelog

Author: Done.

For details on the Azure SDK for Java (October 2019 Preview) release, you can refer to the [release announcement](https://aka.ms/azure-sdk-preview4-java).

This package's
[documentation](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-queue_12.0.0-preview.4/sdk/storage/azure-storage-queue/README.md)
and
[samples](https://github.com/Azure/azure-sdk-for-java/blob/azure-storage-queue_12.0.0-preview.4/sdk/storage/azure-storage-queue/src/samples/java/com/azure/storage/queue)

> - UUpdated to use correct Java Bean getter/setter styles.

Reviewer 2: ```suggestion
- Getters and setters were updated to use Java Bean notation.
```

> - UUpdated to use correct Java Bean getter/setter styles.

Author: Done.

> - Added `getName` for fetching the resource names.

Reviewer 2: Could you expand this is getQueueName

Author: Done.

- Updated to be fully compliant with the Java 9 Platform Module System.
- Changed `VoidResponse` to `Response<Void>` on sync API, and `Mono<VoidResponse>` to `Mono<Response<Void>>` on async API.
- Fixed metadata does not allow capital letter issue. [`Bug 5295`](https://github.com/Azure/azure-sdk-for-java/issues/5295)

> - `getURL` API now returns URL with scheme, host, resource name and snapshot if any.

Reviewer 2: Specify the methods

Author: Done.
